auditor/src/client/mod.rs

```diff
@@ -29,45 +31,106 @@ impl std::fmt::Display for ClientError {
             "{}",
             match self {
                 ClientError::RecordExists => ERR_RECORD_EXISTS.to_string(),
+                ClientError::InvalidTimeout => ERR_INVALID_TIMEOUT.to_string(),
                 ClientError::ReqwestError(e) => format!("Reqwest Error: {e}"),
             }
         )
     }
 }
 
+impl From<reqwest::Error> for ClientError {
+    fn from(error: reqwest::Error) -> Self {
+        ClientError::ReqwestError(error)
+    }
+}
+
+impl From<OutOfRangeError> for ClientError {
+    fn from(_: OutOfRangeError) -> Self {
+        ClientError::InvalidTimeout
+    }
+}
```

Comment:
Nitpick: I would prefer it if `OutOfRangeError` were not imported, and instead is used with the full qualifier: `chrono::OutOfRangeError`. In my opinion, this makes it easier to follow the code.

Reply:
Done